webextension/background/loadSelector.js

```diff
@@ -26,6 +26,7 @@ window.loadSelector = (function () {
         }, () => {
           if (chrome.runtime.lastError
               && chrome.runtime.lastError.message != "Script returned non-structured-clonable data") {
+              catcher.unhandled(chrome.runtime.lastError, {script: script});
```

Comment:
This particular load pattern was the product of some experimentation.  executeScript doesn't always give an error object that matches chrome.runtime.lastError.